docs/plugin-protocol/tfplugin5.proto

```diff
@@ -811,3 +819,45 @@ message CallFunction {
         FunctionError error = 2;
     }
 }
+
+message ListResource {
+    message Request {
+        // type_name is the managed resource type name.
```

Comment:
```suggestion
        // type_name is the list resource type name.
```

While this coincides with the managed resource type, the type name here really is the list resource type name. There could be a meta-list block which never returns resource objects, and is only expected to support list composition. In such instance, there may not be a corresponding managed resource.